specification/synapse/data-plane/Microsoft.Synapse/preview/2019-11-01-preview/roles.json

```diff
@@ -0,0 +1,172 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01-preview",
+    "title": "SynapseClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{workspaceName}.{SynapseDnsSuffix}",
+    "parameters": [
+      {
+        "name": "workspaceName",
+        "description": "The name of the workspace to execute operations on.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      },
+      {
+        "$ref": "#/parameters/SynapseDnsSuffixInPath"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/rbac/roles": {
+      "get": {
+        "tags": [
+          "SynapseRoles"
+        ],
+        "operationId": "Roles_ListRoles",
+        "description": "List roles.",
+        "x-ms-examples": {
+          "Get access control information": {
+            "$ref": "./examples/Roles_ListRoles.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success response.",
+            "schema": {
+              "$ref": "#/definitions/RolesListResponse"
```

Comment:
This operation fits the condition of the x-ms-pageable extension, please use it:
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable

Reply:
> This operation fits the condition of the x-ms-pageable extension, please use it:
> https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable

Added x-ms-pageable